### ISSUE TITLE
Is measurable quantity function f(θ) must be computed from z_m?

### ISSUE THREAD

**Issue by ziw42** [pomonam/kronfluence#25]
Hi! I got a little question about the "compute_measurement()" function in the "Task" class. It seems we must compute this measurable quantity from z_m since we need to use "batch" in this function. However, in the paper you mentioned this f(θ) could be " logits for a query example z_q" (page 10), in which z_q is an independent data point. I would like to ask how should I apply such a f(θ) that has nothing directly to do with z_m, but with an independent data point z_q. I might got some wrong understanding about the code or the paper, and thank you for you notice.

**Comment by pomonam** [comment]
Hi, yes - the measurement is computed with query examples. Although we share the same name for the `batch`, the code will compute the measurement using the `query_dataset` when you compute the pairwise influence scores.
